demo/src/App.js

```diff
@@ -1,15 +1,36 @@
-import React from 'react';
-import {Navbar, Nav, Image} from 'react-bootstrap';
+import React, {useMemo} from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import {Navbar, Nav, Image, Badge} from 'react-bootstrap';
 import {Link} from 'react-router-dom';
 import {Router} from './Router.js';
 import Logo from './logo.svg';
 import './App.css';
 
 /**
- * @return {!JSX} Wrapper component that wraps the entire site with
- *     common components such as a header, navigation, and shopping cart.
+ * Computes the total number of items in the cart.
+ * @param {ItemStore} items A {@link ItemStore} listing of items
+ * to purchase and their desired quantity.
```

Comment:
Nit: Indent 4 spaces for continuation lines on @param documentation.
https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments